src/components/social-nav/index.js

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+import Link from "../link"
+import Icon from "../icon"
+import styles from "./styles.module.scss"
+
+const SocialNav = ({ links }) => (
+  <nav className={styles.social_nav}>
+    {links.map((link, index) => (
+      <Link to={link.url} key={index}>
+        <Icon name={link.icon} />
+      </Link>
+    ))}
+  </nav>
+)
```

Comment:
I'm on the fence about the effectiveness of having this as a top-level component. My gut says to only pull it out as a shared component when it makes sense. But I don't know. Let's assess how much we're going to use it.